### ISSUE TITLE
Contribution request 

### ISSUE THREAD

**Issue by AlDeen20** [imAbdelhadi/handbook-contribute-to-open-source#1]
Hi brother, I'm Essam Aldeen from Iraq, I wanna contribute to the translation of the book.
I've been an SEO Expert and Technical writer for more than 5 years.
I wrote with Arabmarketcap & Arabium and Medium.
I have worked as translator as well
Is there any way to contribute?

**Comment by imAbdelhadi** [comment]
Yes you are welcome to any contribution. Please leave your email/number to make a call as soon.

**Comment by AlDeen20** [comment]
Hi, +9647803998461
essamalshemriy@gmail.com

**Comment by AlDeen20** [comment]
Please if you take the informations delete my comment
